androidenhancedvideoplayer/build.gradle

```diff
@@ -71,14 +71,18 @@ dependencies {
 }
 
 afterEvaluate {
+    def VERSION_NAME = project.properties.get("VERSION_NAME", null).toString()
+    def GROUP_ID = project.properties.get("GROUP").toString()
```

Comment:
Do we need to use the `null` fallback on `VERSIO_NAME`? I'd expect it to be the default behavior returned. Also, I think we should fail instead of publishing with `"null"` as `VERSION_NAME` (Same for `GROUP_ID`)